lib/services_oper.py

```diff
@@ -113,7 +113,8 @@ def process_service(service, app_name, curr_train):
                     port_name=processed_port['port_name'],
                     port=processed_port['port'],
                     protocol=processed_port['protocol'],
-                    train=curr_train
+                    train=curr_train,
+                    note="-" if processed_port.get('note') is None else processed_port['note']
```

Comment:
```suggestion
                    note=processed_port.get('note', '-')
```
Isn't that the same?